lambda/subArea/DELETE/index.js

```diff
@@ -125,6 +126,27 @@ async function deleteSubAreaRecords(subAreaId, orcs, context) {
   return response.Attributes?.activities.SS;
 }
 
+async function archiveSubAreaRecords(subAreaId, orcs, context) {
```

Comment:
nit: We should standardize our nomenclature at some point. A lot of our code has the term `record` which can refer to an `activityRecord` (db item that holds monthly data entries for a subarea's activities) or a `subareaRecord` (db item for subarea). We should decide if `record` is synonymous with 'database item' or whether it is synonymous with the particular `activityRecord` database items. Personally I would change this function name to `archiveSubarea` since it only does 1 subarea at a time.

Reply:
Good idea - fixed in 774a408